app/models/choice.rb

```diff
@@ -2,6 +2,8 @@ class Choice < ActiveRecord::Base
   belongs_to(:answer, :inverse_of => :choices, :touch => true)
   belongs_to(:option, :inverse_of => :choices)
```

Comment:
We are slowly moving toward newer syntax. As a general rule, new and refactored code should use new syntax, old stuff should be left alone.